src/icons/Vector.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="16" viewBox="0 0 18 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
what is this for? and why the uppercase V ?

Reply:
it's icon like. But now it's unused. I remove this icon and move the default image to the image component in the last commit

Reply:
as i see it's deleted. can be resolved